provision.sh

```diff
@@ -61,3 +61,11 @@ sudo apt-add-repository ppa:brightbox/ruby-ng
 sudo apt-get update
 sudo apt-get -y install ruby2.3
 sudo gem install bundler
+
+echo "[PROVISIONER] Installing CMAKE"
+cd ~
+wget https://cmake.org/files/v3.10/cmake-3.10.0-rc5.tar.gz
+cd ~/cmake-3.10.0-rc5
```

Comment:
Do this in the `tmp` directory. That way we will avoid unused files in the home path.

Reply:
Should I do the same for node, elixir, firefox and hub?

Reply:
Yes, that would be a good idea.

Or, alternatively, we can add as `cd /tmp` the first line in the provision script. That way, tmp will be the default.